openwisp_monitoring/device/tests/test_models.py

```diff
@@ -60,7 +60,7 @@ class BaseTestCase(DeviceMonitoringTestCase):
                 },
                 "wireless": {
                     "channel": 1,
-                    "clients": {},
+                    "clients": [],
```

Comment:
Please don't leave it empty. Instead add real data so that the testing is more useful for what you have implemented. You can just copy-paste what Federico posted here -> https://github.com/openwisp/openwisp-monitoring/issues/60#issuecomment-625617960